CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added configuration for token TTL
   [cyberark/conjur#2510](https://github.com/cyberark/conjur/pull/2510)
 
+### Fixed
+- Memory leak in postgres by active conjur connections. Now active connections are closed on policy creations and updations.
```

Comment:
We should provide a little more details throughout this PR to help answer the "why" question for others in the future.

Specifically for the changelog, I would suggest something like:

```md
- Previously, the temporary schemas used to modify Conjur policy
  caused the Postgres database cache to leak memory over time,
  leading to an eventual crash. Now, we recycle the database
  connection after modifying policy to free this cache and prevent
  the memory leak from occurring.
  [Link to PR...]()
```